_pages/general/visas.md

```diff
@@ -5,17 +5,89 @@ description: How to obtain a visa, useful resources and FAQ's relating to visas
 
 # Visas
 
-If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration
-department before you commence our recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the
-482 visa.
+If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration department before you commence a recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the 482 visa.
 Once your visa is approved, you can get started setting up things you'll need for life in Australia. For more
-information on 482 visas, please visit the FAQs section.
+information on 482 visas.
 
-## How long does the application process take?
+## FAQ's
+
+### What type of visa should I apply for?
+
+**Skilled Independent (Subclass 189)**
+
+If you are a qualified professional or trades person with work experience that will benefit the Australian economy, and your occupation is on Medium and Long-term Strategic Skills List (MLTSSL), this visa is for you.
+
+In addition to meeting the basic requirements mentioned above, you are also required to pass a points test with points being awarded for age, English language ability, specific work experience, qualification, spouse skills, Australian skilled work experience, Australian qualifications and fluency in one of Australia's major community languages.
+
+The current passmark for this particular visa category is 60 points however, because of ceilings imposed on certain occupations, you may in fact be required to obtain more than 60 points in order to receive an invitation to apply for this visa.
```

Comment:
The minimum points changed to 65 earlier this year.
Is better to say `as the time of this writing` and add a link to the correct page.
There's also a page showing the minimum scores on last round. (https://immi.homeaffairs.gov.au/visas/working-in-australia/skillselect/invitation-rounds)

Reply:
changed wording completely so we dont have to specify points 7ea35071100cc60f45aeb4c4fd1a3dc44590f012